.buildkite/packaging.pipeline.yml

```diff
@@ -172,6 +174,73 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "SNAPSHOT: {{matrix}} FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
+          SNAPSHOT: true
+          # packaging with `DEV=true` may cause linker issues while crosscompiling https://github.com/elastic/beats/issues/41270
+          DEV: false
+          FIPS: true
+        command: ".buildkite/scripts/packaging/package-dra.sh {{matrix}}"
+        agents:
+          provider: gcp
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+
+      - label: "SNAPSHOT: {{matrix}} Linux/arm64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_ARM_FIPS}"
+          SNAPSHOT: true
+          # packaging with `DEV=true` may cause linker issues while crosscompiling https://github.com/elastic/beats/issues/41270
+          DEV: false
+          FIPS: true
+        command: ".buildkite/scripts/packaging/package-dra.sh {{matrix}}"
+        agents:
+          provider: "aws"
+          imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+          - x-pack/agentbeat
+
+      ## Agentbeat needs more CPUs because it builds many other beats
+      - label: "SNAPSHOT: x-pack/agentbeat all artifacts apart from linux/arm64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
```

Comment:
But agentbeat is on the arm64 matrix. Is it because the the arm already have the power to build the agent beat? Perhaps could add a comment there explaining it. Or move this section immediately after `"SNAPSHOT: {{matrix}} FIPS"` so it's easier to see where the agentbeat x86 is

Reply:
On arm64 the machine is powerful enough to build also agentbeat (it's the same for non-FIPS package steps)
Here I mirrored the structure of the existing pipeline for non-FIPS package steps.
I have no objections about moving things around but I would then move also the non-FIPS steps to keep everything consistent.

@pazone what do you think about reordering the steps in the pipeline YAML ? (Execution order will not change AFAICS)

Reply:
either is fine, at first it seemed there was the agent beat missing. It's not a blocker. I'll resolve the comment